kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -91,7 +91,7 @@
               class="login-btn"
               :text="$tr('nextLabel')"
               :primary="true"
-              :disabled="busy"
+              :disabled="!isNextValid"
```

Comment:
The button should be disabled if `busy` or `!isNextValid`
```suggestion
              :disabled="busy || !isNextValid"
```